style.css

```diff
@@ -1,20 +1,117 @@
-*{
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-    height: 100vh;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
-.box{
-    background-color: blueviolet;
-    border: 1px solid black;
-    width: 200px;
-    height: 50vh;
-    flex: 1;
+
+@media only screen and (max-width: 768px) {
+  body {
+    height: 100vh;
+    width: 100%;
+    left: 0;
+    top: 0;
+    background-color: #fff;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 4vh 5vh;
+  }
+
+  .top-logo h2 {
+    font-family: 'Poppins', sans-serif;
+    font-size: 22px;
```

Comment:
- [ ] Please use appropriate font-size for heading `<h1>`, paragraph`<p>` and 'Let's connect' `<a>` to match the font-size of [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=RfqrWNFQxQ6al11h-0) as mentioned in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

> Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

**_Your design_**

<img width="264" alt="Screenshot 2023-02-16 at 11 35 23 AM" src="https://user-images.githubusercontent.com/44798044/219299464-d096d743-3a69-4118-a87c-bb97e6d43a74.png">

_**Figma design**_

![Uploading Screenshot 2023-02-16 at 11.18.47 AM.png…]()